ros/src/util/packages/autoware_launcher/documents/plugin/create.md

```diff
@@ -0,0 +1,39 @@
+
+## Create Plugin File
+
+If you want to use your launch file. Create the plugin file. For example, you can add ndt matching as following.<br>
+
+1. Create plugin file template by text editor (e.g. autoware_launcher/plugins/refs/ndt_matching.yaml).
+```
+    format: Autoware Launcher Plugin Version 0.1
+    rosxml: $(find lidar_localizer)/launch/ndt_matching.launch
+```
+1. Launch plugin file generate tool.<br>
+`rosrun autoware_launcher tool refs/ndt_matching`
+1. Set type and default value.<br>
+![plugin_tool](./plugin_tool.png)
+1. Push export button. The tool will set the text to clipboard. Paste it to yaml file.<br>
```

Comment:
```suggestion
1. Once configuration is set, press the *Export* button below. This will copy the configuration to the clipboard. Open again the plugin file and paste the new configuration.<br>
```